spynnaker/pyNN/models/spike_source/spike_source_poisson_vertex.py

```diff
@@ -230,30 +359,8 @@ def create_machine_vertex(
             constraints, label)
 
     @property
-    def rate(self):
-        return self._rate
-
-    @rate.setter
-    def rate(self, rate):
-        self._rate = utility_calls.convert_param_to_numpy(rate, self._n_atoms)
-
```

Comment:
Isn't it supposed to be possible to do e.g. population.set(rate=100) from a top-level PyNN script?  Doesn't this remove that possibility?  (see e.g. https://github.com/SpiNNakerManchester/sPyNNaker8/blob/master/p8_integration_tests/test_change_neuron_parameters_between_runs/test_change_parameter.py)